src/test/resources/features/personalInformation/edit_personalInformation.feature

```diff
@@ -0,0 +1,17 @@
+@PersonalInformation
+Feature: Edit Personal Information
+
+  Background:
+    Given I log in to Salesforce with a valid Account
+
+  @restorePersonalInformation
+  Scenario: Update personal information with the minimum values
+    When I go to "PERSONAL INFORMATION PAGE"
+    And I verify that the title of the panel is "Personal Information"
```

Comment:
I think this step is unnecesary, we don't need to verify that, please delete it.